styles/styles.css

```diff
@@ -208,3 +138,91 @@ main .section.light,
 main .section.highlight {
   background-color: var(--light-color);
 }
+
+div, span, strong, p, a, i {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+h3 {
+  font-weight: 600;
```

Comment:
Please use the global tokens that you already propose to add in this pr.

Reply:
The token definition will be done by https://github.com/aemsites/icicidirect/issues/27

Reply:
updated.